rocAL/rocAL/source/meta_data/meta_node_resize_crop_mirror.cpp

```diff
@@ -37,10 +37,10 @@ void ResizeCropMirrorMetaNode::update_parameters(pMetaDataBatch input_meta_data,
     {
         _batch_size = input_meta_data->size();
```

Comment:
Where are we handling the resize case here?

Reply:
Unlike other metanodes where resize case is handled by getting the output ROI info and rescaling the metadata accordingly, for resize_crop_mirror and crop_resize, the resize width and height is set as the max width and height of the output tensor which is fetched (L42-43) and used to calculate the scaling factor(L68-69)